ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -829,7 +829,15 @@ class OverflowAvatarView: AvatarView {
 extension AvatarView: UIPointerInteractionDelegate {
     @available(iOS 13.4, *)
     public func pointerInteraction(_ interaction: UIPointerInteraction, styleFor region: UIPointerRegion) -> UIPointerStyle? {
-        let pointerEffect = UIPointerEffect.lift(.init(view: self))
-        return UIPointerStyle(effect: pointerEffect, shape: nil)
+        if let superview = window {
```

Comment:
Use guard .. else for early returns?
https://github.com/microsoft/swift-guide/blob/gh-pages/EarlyReturn.md

guard let superview = window else {
        return nil
 }